multiqc/modules/genomescope2/genomescope2.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from genomescope """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re, base64
+from multiqc.plots import table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """Genomescope2 module"""
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="Genomescope2",
+            anchor="Genomescope2",
+            href="https://github.com/tbenavi1/genomescope2.0",
+            info="Reference-free profiling of polyploid genomes.",
+            doi="https://doi.org/10.1038/s41467-020-14998-3",
+        )
+
+        # Find and load any GENOMESCOPE reports
+        self.summary_data = dict()
+        for f in self.find_log_files("genomescope2/summary"):
+            self.parse_summary_log(f)
+
+        log.info("Found {} summary reports".format(len(self.summary_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.summary_data, "multiqc_genomescope_summary")
+
+        self.add_section(name="Summary", anchor="genomescope-summary", plot=self.summary_table())
```

Comment:
A description and help text are good to add if at all possible. Just a short one-liner of what the plot is, then if you can some help text explaining how to interpret it (again fine to copy from the tool docs where appropriate).